oaknational/Oak-Web-Application
# Feat/lesq 268/update search component [LESQ-268]

## Description

Discussed with Monika and decided to keep the changes as small as possible, which is:
- placeholder text in the search box is now 'Search by keyword or topic'
- icon updated to search icon instead of go

I updated the placeholder text to be a prop for the search component, as I thought this component was used elsewhere  and would need different copy depending on the situation but looking through I'm not actually sure this is the case - so happy to revert that if it makes more sense to keep it harcoded within the component.

## How to test

1. Go to https://deploy-preview-1901--oak-web-application.netlify.thenational.academy/beta/teachers-2023
2. You should see the updated copy and icon for the search box

## Screenshots

How it used to look:
<img width="400" alt="Screenshot 2023-09-13 at 11 25 55" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/996d22bd-27d0-4008-9f37-1f3ee84aec05">
  
How it should now look:
<img width="400" alt="Screenshot 2023-09-13 at 11 27 44" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/e7d36a54-a488-4ab0-aac3-f60126e7fc4a">


## ACs
- [ ]  Change dummy text to: Search by keyword or topic
- [ ]  Change icon from ‘Go’ to ‘Search’
